components/table/Table.tsx

```diff
@@ -855,6 +856,20 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
     return title;
   }
 
+  getColumnTitle: any = (node: any, parentNode: any) => {
+    if(!node) {
+      return;
+    }
+    if (typeof node !== 'string') {
```

Comment:
https://github.com/ant-design/ant-design/blob/master/components/table/interface.tsx#L13
It maybe a `Number` or `Function`.

Reply:
done